libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.cpp

```diff
@@ -389,6 +389,11 @@ bool ESP8266WiFiGenericClass::mode(WiFiMode_t m) {
 
     bool ret = false;
 
+    if (m == WIFI_OFF)

```

Comment:
Instead of testing for OFF, should this be testing for !STA && !STA_AP?

Reply:
`!STA && !STA_AP` == `OFF || AP`, so yes it is better.